config/quality.gradle

```diff
@@ -55,6 +55,7 @@ android {
         abortOnError true
         xmlReport true
         htmlReport true
+        checkGeneratedSources true
```

Comment:
What was this needed for?

Reply:
I used this as otherwise `./gradlew lintDebug` fails and it gives an error for the unused resource (`geo_widget_answer.xml`)

Reply:
Ah do you mean `rating_widget_answer.xml`? I was confused initially but guessing this is because of View Binding's magic. Makes sense now!